lib/src/test/java/com/cloudbees/groovy/cps/CpsTransformerTest.java

```diff
@@ -172,16 +174,19 @@ public void workflowCallingWorkflow() throws Throwable {
      */
     @Test
     public void exceptionFromNonCpsCodeShouldBeCaughtByCatchBlockInCpsCode() throws Throwable {
-        assertEvaluate("String index out of range: -2",
-            "def foo() {\n" +
-            "  'abc'.substring(5);\n" + // will caught exception
-            "  return 'fail';\n" +
-            "}\n" +
-            "try {\n" +
-            "  return foo();\n" +
-            "} catch(StringIndexOutOfBoundsException e) {\n" +
-            "  return e.message;\n" +
-            "}\n");
+        try {
+            evalCPS(
+                "def foo() {\n" +
+                "  'abc'.substring(5);\n" + // will caught exception
+                "  return 'fail';\n" +
+                "}\n" +
+                "return foo()\n");
+            fail("Should have failed");
+        } catch(StringIndexOutOfBoundsException e) {
```

Comment:
Oh I guess this test specifically wanted to check the behavior of try/catch, so I will refactor this.

Reply:
https://github.com/cloudbees/groovy-cps/pull/120/commits/c1a2ff4c70527eb315a75c1946a7f0fe460d9c06